tests/rptest/services/redpanda.py

```diff
@@ -992,6 +992,60 @@ def start_rp():
         self.start_service(node, start_rp)
         self._started.append(node)
 
+    def start_node_with_rpk(self, node, additional_args=""):
+        """
+        Start a single instance of redpanda using rpk. similar to start_node, 
+        this function will not return until redpanda appears to have started 
+        successfully.
+        """
+        node.account.mkdirs(RedpandaService.DATA_DIR)
+        node.account.mkdirs(os.path.dirname(RedpandaService.NODE_CONFIG_FILE))
+
+        def rpk_start():
+            # Pass environment variables via FOO=BAR shell expressions
+            env_preamble = " ".join(
+                [f"{k}={v}" for (k, v) in self._environment.items()])
+
+            cmd = (f"{env_preamble} {self.find_binary('rpk')} redpanda start"
```

Comment:
we can probably place this entire method nicely in the `rpk` class in `rptest/clients`